test/app-luatest/gh_7479_bug_fix_uri_lib_test.lua

```diff
@@ -0,0 +1,49 @@
+local t = require('luatest')
+local g = t.group()
+local popen   = require('popen')
+local net_box = require('net.box')
+local server  = require('test.luatest_helpers.server')
+
+-- Create test instance.
+g.before_all = function()
+    g.server = server:new({alias = 'test-gh-7479'})
+    g.server:start()
+    -- Get port for connection testing.
+    g.server_port = g.server:eval("return require('console').listen(0):name()['port']")
```

Comment:
Please use `g.server:exec()` instead of eval.

Also, please keep all lines within 80 characters.

Reply:
Done